english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md

```diff
@@ -0,0 +1 @@
+Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
```

Comment:
I've given up.I really don't know what to do other than deleting the old link and replaced it with the new one that is reachablefrom the change branch in the marshmallow.md file. 

Your intensive care is needed.
Thanks'Hisham Showeir
 


     On Wednesday, June 10, 2015 2:35 AM, Caroline Buckey <notifications@github.com> wrote:
   

 In english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md:> @@ -0,0 +1 @@
> +Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
My point is that I can still see the old link - ../breakfast/breakfast.md. I can see both this link, and your fixed link, so I believe you need to remove this one.—
Reply to this email directly or view it on GitHub.